offchain/dispatcher/Cargo.toml

```
@@ -13,6 +13,7 @@ contracts = { path = "../contracts" }
 http-server = { path = "../http-server" }
 rollups-events = { path = "../rollups-events" }
 types = { path = "../types" }
+log = {path = "../log"} 
```

Just keeping them in alphabetic order:

```suggestion
log = { path = "../log" }
rollups-events = { path = "../rollups-events" }
types = { path = "../types" }
```

---

Done at 9e1b6f8e7bc477c334f4e109cf3bb7a1642bc44d